doc/classes/LineEdit.xml

```diff
@@ -145,19 +145,20 @@
 			<argument index="1" name="to" type="int" default="-1">
 			</argument>
 			<description>
-				Select the text inside [code]LineEdit[/code] by the given character positions. [code]from[/code] is default to the beginning. [code]to[/code] is default to the end.
+				Selects characters inside [LineEdit] between [code]from[/code] and [code]to[/code]. By default [code]from[/code] is at the beginning and [code]to[/code] at the end.
 				[codeblock]
-				select()     # select all
-				select(5)    # select from the fifth character to the end.
-				select(2, 5) # select from the second to the fifth character.
+				$LineEdit.text = "Welcome"
+				select()     # Welcome
+				select(4)    # ome
+				select(2, 5) # lco
```

Comment:
If the LineEdit node is named "LineEdit", you need to use `$LineEdit.select()`, or change to `text = "Welcome"`